docs/code-of-conduct.rst

```diff
@@ -0,0 +1,33 @@
+Code of Conduct
+===============
+
+Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation.
+
```

Comment:
Per this discussion: https://github.com/writethedocs/www/pull/115/files/ef1be4587e35ad99a53575591c92ef8508ad1142#r95243668, 

I propose changing the start of this para to:

While diversity is one of our strengths, it can lead to communication issues. To that end, we have a few ground rules...